ide/static/js/content.js

```diff
@@ -680,6 +683,47 @@ class Content extends React.Component {
     this.modalContent = <ModelZoo importNet={this.importNet}/>;
     this.openModal();
   }
+  
+  handleClick(event) {
+    this.placeholder = false;
+    event.preventDefault();
+
+    const net = this.state.net;
+    const id = event.target.id;
+    const prev = net[`l${this.state.nextLayerId-1}`];
+    const next = data[id];
+    const zoom = instance.getZoom();    
+    const layer = {};
+    let phase = this.state.selectedPhase;
+    
```

Comment:
Need to add a case to handle situation when there is no layer, can do something like this
```
// To handle case when no layer has been added
    if (Object.keys(net).length==0){
      layer.connection = { input: [], output: [] };
      layer.info = {
            type: id.toString(),
            phase,
            class: ''
          }
      layer.params = {'endPoint': [next['endpoint'], false]};        
      Object.keys(next.params).forEach(j => {
        layer.params[j] = [next.params[j].value, false];
      });    
      layer.props = JSON.parse(JSON.stringify(next.props))
      const height = Math.round(0.05*window.innerHeight, 0);
      const width = Math.round(0.35*window.innerWidth, 0);
      var top = height + Math.ceil(41-height);
      var left = width;
      layer.state = {
            top: `${top}px`,
            left: `${left}px`,
            class: '' 
          }
      layer.props.name = `${next.name}${this.state.nextLayerId}`;          
      this.addNewLayer(layer);
      this.clicker = true;
    }
```